lib/astral_plane/http_client/abstract_http_client.rb

```diff
@@ -24,8 +24,30 @@ def self.get(base_url, endpoint, token)
     http = Net::HTTP.new(uri.host, uri.port)
     request = Net::HTTP::Get.new(uri.request_uri)
     request['Authorization'] = "Bearer: #{token}"
+    request['content-type'] = "application/json"
```

Comment:
Would you mind following the paradigm of headers being Capitaliezd? `Content-Type`